internal/code-generator/generator/linguist-commit.go

```diff
@@ -0,0 +1,17 @@
+package generator
+
+import (
+	"bytes"
+	"text/template"
+)
+
+// Commit takes in commit and builds source file from tmplPath. It's comply with type File signature.
```

Comment:
* s/`Commit takes in commit and builds source file`/`Commit takes a commit and builds the source file`/?
* `It's comply` sounds incorrect. Maybe you meant 'It complies'?